.circleci/config.yml

```diff
@@ -216,60 +216,20 @@ commands:
           keys:
             - 'libsodium-fork-{{ .Environment.CIRCLE_STAGE }}-{{ checksum "tmp/libsodium.md5" }}'
 
-  generic_shorttest:
-    description: Run short tests from build workspace, for re-use by diferent architectures
+  generic_buildtest:
+    description: Run build tests from build workspace, for re-use by diferent architectures
     parameters:
       circleci_home:
         type: string
         default: "/home/circleci"
       result_subdir:
         type: string
-    steps:
-      - attach_workspace:
-          at: << parameters.circleci_home >>
-      - run: mkdir -p /tmp/results/<< parameters.result_subdir >>
-      - restore_cache:
-          keys:
-            - 'go-cache-{{ .Environment.CIRCLE_STAGE }}-'
-      - run:
-          name: Run short tests
-          no_output_timeout: 30m
-          command: |
-            set -e
-            set -x
-            export PATH=$(echo "$PATH" | sed -e 's|:<< parameters.circleci_home >>/\.go_workspace/bin||g' | sed -e 's|:/usr/local/go/bin||g')
-            export KMD_NOUSB=True
-            export GOPATH="<< parameters.circleci_home >>/go"
-            export PATH="${PATH}:${GOPATH}/bin"
-            export ALGORAND_DEADLOCK=enable
-            GOLANG_VERSION=$(./scripts/get_golang_version.sh)
-            eval "$(~/gimme "${GOLANG_VERSION}")"
-            scripts/configure_dev.sh
-            scripts/buildtools/install_buildtools.sh -o "gotest.tools/gotestsum"
```

Comment:
Was the vm setup stuff here unnecessary?

Reply:
I'm not sure, I kept what was already in the file

Reply:
Ohh i see, the diff view just doesn't show that it's still in the command config from the duplicate command

Reply:
Oh that's what you meant! Yeah that tripped me up when I first looked at the diff too.